src/components/by-user-view.jsx

```diff
@@ -11,95 +11,89 @@ import IssueList from './issue-list';
+        card={card}
+        columnRegExp={columnRegExp}
+        />
     );
-
-  }
-});
-
-
-const UsersView = React.createClass({
-  render() {
-    const {repoInfos, columnData, cards, columnRegExp} = this.props;
-    const [{repoName}] = repoInfos; // primaryRepoName
-
-    let sortedCards = FilterStore.filterAndSort(cards, true);
-
-    let kanbanColumnCount = 0; // Count the number of actual columns displayed
-
-    const kanbanColumns = _.map(columnData, (login) => {
-      // If we are filtering by a kanban column then only show that column
-      // Otherwise show all columns
-      const columnCards = _.filter(sortedCards, (card) => {
-        return card.issue.owner && card.issue.owner.login === login || card.issue.user.login === login;
-      });
+  });
+
+  const heading = (
+    <span className='user-title'>
+      <UserIcon/>
+      {login}
+    </span>
+  );
+
+  return (
+    <div key={login} className='kanban-board-column'>
+      <IssueList title={heading}>
+        {issueComponents}
+      </IssueList>
+    </div>
+  );
+
+}
+
+function UsersView(props) {
+  const {repoInfos, columnData, cards, columnRegExp} = props;
+  const [{repoName}] = repoInfos; // primaryRepoName
+
+  let sortedCards = FilterStore.filterAndSort(cards, true);
+
+  let kanbanColumnCount = 0; // Count the number of actual columns displayed
+
+  const kanbanColumns = _.map(columnData, (login) => {
+    // If we are filtering by a kanban column then only show that column
+    // Otherwise show all columns
+    const columnCards = _.filter(sortedCards, (card) => {
+      return card.issue.owner && card.issue.owner.login === login || card.issue.user.login === login;
+    });
 
 
-      // Show the column when:
-      // isFilteringByColumn = label (the current column we are filtering on)
-      // !isFilteringByColumn && (!getShowEmptyColumns || columnCards.length)
-
-      kanbanColumnCount++; // Count the number of actual columns displayed
-      /*HACK: Column should handle milestones */
-      return (
-        <KanbanColumn
-          key={kanbanColumnCount}
-          login={login}
-          cards={columnCards}
-          primaryRepoName={repoName}
-          columnRegExp={columnRegExp}
-        />
-      );
-    });
+    // Show the column when:
+    // isFilteringByColumn = label (the current column we are filtering on)
+    // !isFilteringByColumn && (!getShowEmptyColumns || columnCards.length)
 
+    kanbanColumnCount++; // Count the number of actual columns displayed
     return (
-      <div className='kanban-board'>
-        {kanbanColumns}
-      </div>
+      <KanbanColumn
+        key={kanbanColumnCount}
+        login={login}
+        cards={columnCards}
+        primaryRepoName={repoName}
+        columnRegExp={columnRegExp}
+      />
     );
-  }
-});
+  });
+
+  return (
+    <div className='kanban-board'>
+      {kanbanColumns}
+    </div>
+  );
+}
 
+class RepoKanbanShell extends Component {
+  static displayName = 'RepoKanbanShell';
```

Comment:
Another unecessary `displayName`

Reply:
Seems like I didn't make it clear enough to check all `displayName` :(

Reply:
You did say that, but I forgot to check all of them :(